radixdlt/src/test/java/com/radixdlt/consensus/AddressBookGenesisBFTHeaderProviderTest.java

```diff
@@ -36,7 +35,7 @@ public void when_quorum_size_is_one__then_should_emit_self() {
 		ECPublicKey self = mock(ECPublicKey.class);
 		when(self.euid()).thenReturn(EUID.ONE);
 		AddressBook addressBook = mock(AddressBook.class);
-		AddressBookGenesisHeaderProvider validatorSetProvider = new AddressBookGenesisHeaderProvider(
+		AddressBookGenesisValidatorSetProvider validatorSetProvider = new AddressBookGenesisValidatorSetProvider(
```

Comment:
This isn't used anymore.

Reply:
Actually, this whole test looks incomplete now that the assertion has been removed.